app/javascript/packs/articleModerationTools.js

```diff
@@ -12,20 +13,16 @@ const initializeModerationsTools = async () => {
   const { initializeFlagUserModal } = await import('./flagUserModal');
 
   // article show page
-  if (
-    user?.trusted &&
-    user?.id !== articleAuthorId &&
-    !top.document.location.pathname.endsWith('/mod')
-  ) {
-    initializeActionsPanel(user, path);
-    initializeFlagUserModal(articleAuthorId);
-    // dev.to/mod
-  } else if (user?.trusted && top.document.location.pathname.endsWith('/mod')) {
-    initializeActionsPanel(user, path);
-    initializeFlagUserModal(articleAuthorId);
+  if (user?.trusted) {
+    if (user?.id !== articleAuthorId && !isModerationPage()) {
```

Comment:
I've extracted the logic to check if we are on the moderation page into a small function so it can be reused in a few places.